package.json

```diff
@@ -91,12 +93,20 @@
     "src/lib/generate.js"
   ],
   "eslintConfig": {
-    "extends": [
-      "@wfcd"
-    ],
+    "root": true,
+    "extends": "@wfcd/eslint-config/esm",
+    "parser": "@babel/eslint-parser",
+    "parserOptions": {
+      "sourceType": "module",
+      "ecmaVersion": 6,
+      "ecmaFeatures": {
+        "modules": true
+      }
+    },
     "rules": {
       "no-underscore-dangle": "off",
-      "no-restricted-syntax": "off"
+      "no-restricted-syntax": "off",
+      "import/prefer-default-export": "off"
```

Comment:
What about we upstream this rule?

Reply:
that's overkill, given that it's generally preferrable in other projects to use default export.